core/test/XiosFile_test.cpp

```diff
@@ -102,42 +95,47 @@ MPI_TEST_CASE("TestXiosFile", 2)
-    REQUIRE(xiosHandler.getFileSplitFreq("day").seconds() == 2 * 24 * 60 * 60);
-    xiosHandler.createFile("hour");
-    xiosHandler.setFileOutputFreq("hour", Duration("P0-0T01:00:00"));
-    xiosHandler.setFileSplitFreq("hour", Duration("P0-0T02:00:00"));
-    REQUIRE(xiosHandler.getFileOutputFreq("hour").seconds() == 60 * 60);
-    REQUIRE(xiosHandler.getFileSplitFreq("hour").seconds() == 2 * 60 * 60);
-    xiosHandler.createFile("minute");
-    xiosHandler.setFileOutputFreq("minute", Duration("P0-0T00:01:00"));
-    xiosHandler.setFileSplitFreq("minute", Duration("P0-0T00:02:00"));
-    REQUIRE(xiosHandler.getFileOutputFreq("minute").seconds() == 60);
-    REQUIRE(xiosHandler.getFileSplitFreq("minute").seconds() == 2 * 60);
-    xiosHandler.createFile("second");
-    xiosHandler.setFileOutputFreq("second", Duration("P0-0T00:00:01"));
-    xiosHandler.setFileSplitFreq("second", Duration("P0-0T00:00:02"));
-    REQUIRE(xiosHandler.getFileOutputFreq("second").seconds() == 1);
-    REQUIRE(xiosHandler.getFileSplitFreq("second").seconds() == 2);
+    xiosHandler.createFile("unittest_year");
+    xiosHandler.setFileOutputFreq("unittest_year", Duration("P1-0T00:00:00"));
+    xiosHandler.setFileSplitFreq("unittest_year", Duration("P2-0T00:00:00"));
+    REQUIRE(xiosHandler.getFileOutputFreq("unittest_year").seconds() == 365 * 24 * 60 * 60);
+    REQUIRE(xiosHandler.getFileSplitFreq("unittest_year").seconds() == 2 * 365 * 24 * 60 * 60);
+    xiosHandler.createFile("unittest_day");
+    xiosHandler.setFileOutputFreq("unittest_day", Duration("P0-1T00:00:00"));
+    xiosHandler.setFileSplitFreq("unittest_day", Duration("P0-2T00:00:00"));
+    REQUIRE(xiosHandler.getFileOutputFreq("unittest_day").seconds() == 24 * 60 * 60);
+    REQUIRE(xiosHandler.getFileSplitFreq("unittest_day").seconds() == 2 * 24 * 60 * 60);
+    xiosHandler.createFile("unittest_hour");
+    xiosHandler.setFileOutputFreq("unittest_hour", Duration("P0-0T01:00:00"));
+    xiosHandler.setFileSplitFreq("unittest_hour", Duration("P0-0T02:00:00"));
+    REQUIRE(xiosHandler.getFileOutputFreq("unittest_hour").seconds() == 60 * 60);
+    REQUIRE(xiosHandler.getFileSplitFreq("unittest_hour").seconds() == 2 * 60 * 60);
+    xiosHandler.createFile("unittest_minute");
+    xiosHandler.setFileOutputFreq("unittest_minute", Duration("P0-0T00:01:00"));
+    xiosHandler.setFileSplitFreq("unittest_minute", Duration("P0-0T00:02:00"));
+    REQUIRE(xiosHandler.getFileOutputFreq("unittest_minute").seconds() == 60);
+    REQUIRE(xiosHandler.getFileSplitFreq("unittest_minute").seconds() == 2 * 60);
+    xiosHandler.createFile("unittest_second");
+    xiosHandler.setFileOutputFreq("unittest_second", Duration("P0-0T00:00:01"));
+    xiosHandler.setFileSplitFreq("unittest_second", Duration("P0-0T00:00:02"));
+    REQUIRE(xiosHandler.getFileOutputFreq("unittest_second").seconds() == 1);
+    REQUIRE(xiosHandler.getFileSplitFreq("unittest_second").seconds() == 2);
```

Comment:
Also recommend parameterizing these strings.

Reply:
Done in e78c7e9090c9b7ecd9ab9e60c045525aa228053c.